library_generation/repo-level-postprocess/readme_update.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# This script should run as a part of new-library generation process.
```

Comment:
I expect README.md to be generated from template.

Reply:
> I expect README.md to be generated from template.

I agree that this should be done, but should we do it in this PR since the goal of this PR is migration?

Reply:
Would you elaborate why this migration needs to change how README.md is generated?

Reply:
As discussed in chat, this is how README.md modified by new-client.py in google-cloud-java.

Reply:
Try to change the template of README.md to remove this script.

Reply:
The template of `README.md` has release-please annotation and this script is no longer needed.